### PR TITLE
Add rules from Egregor ransomware DLL

### PR DESCRIPTION
**Pull request by recvfrom**
Let me know if you'd like me to open tickets corresponding to these two new rules.

Test sample: https://github.com/fireeye/capa-testfiles/pull/86

Thanks!

### CODE REVIEW

File: host-interaction/process/terminate/terminate-process.yml [hunk 1]
```diff
@@ -8,10 +8,12 @@ rule:
       - Process::Terminate Process [C0018]
     examples:
       - C91887D861D9BD4A5872249B641BC9F9:0x401A77
+      - 9B7CCAA2AE6A5B96E3110EBCBC4311F6:0x10010307
   features:
     - and:
       - optional:
         - api: kernel32.OpenProcess
+        - api: ntdll.NtOpenProcess
```
Comment (williballenthin) [inline host-interaction/process/terminate/terminate-process.yml:16]: nice catch

File: host-interaction/os/users/get-logon-sessions.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+rule:
+  meta:
+    name: get logon sessions
+    namespace: host-interaction/os/users
```
Comment (williballenthin) [inline host-interaction/os/users/get-logon-sessions.yml:4]: we have `host-interaction/session` already. perhaps that's a better existing namespace for this rule?

**Review by williballenthin — CHANGES_REQUESTED** [review]
one namespace suggestion, otherwise looks great. thanks @recvfrom!

File: host-interaction/os/users/get-logon-sessions.yml (hunk 2 above)
Comment (williballenthin) [reply host-interaction/os/users/get-logon-sessions.yml:4]: "when to use 'session' and when to use 'users'?" i don't know, but lets go for consistency right now :-)

**Comment by williballenthin** [comment]
> Let me know if you'd like me to open tickets corresponding to these two new rules.

no need! we really don't want process to get in the way of progress. let us know if there's any way we can make contributions even easier

**Review by williballenthin — APPROVED** [review]
thank you!

File: host-interaction/process/terminate/terminate-process.yml (hunk 1 above)
Comment (mr-tz) [reply host-interaction/process/terminate/terminate-process.yml:16]: for reference, there's a `open process` library rule

**Review by mr-tz — COMMENTED** [review]
thank you!

**Pull request merged**